security-oauth2/src/test/groovy/io/micronaut/security/oauth2/client/clientcredentials/ClientCredentialsConcurrentSpec.groovy

```diff
@@ -156,22 +159,54 @@ class ClientCredentialsConcurrentSpec extends Specification {
             this.tokenExpiration = tokenExpiration
         }
 
+
+        @Introspected
+        static class ClientCredentialsForm {
```

Comment:
I had to extract this POJOs to make the test pass with serde 4

Reply:
I created an issue in serde https://github.com/micronaut-projects/micronaut-serialization/issues/441